recipe/meta.yaml

```diff
@@ -19,26 +19,27 @@ build:
 requirements:
   host:
     - pip
-    - python >=3.7
+    - python >=3.9
     - setuptools-scm
   run:
     - magicgui >=0.3.7
-    - napari >=0.4.15
+    - napari >=0.4.17
+    - npe2 >=0.1.2
     - numpy
-    - python >=3.7
+    - python >=3.9
     - scikit-image >=0.19.2
     - toolz
 
 test:
-  imports:      # [not linux]
+  imports:   # [not linux]
     - affinder  # [not linux]
   commands:
     - pip list
     - pip check
     - export QT_DEBUG_PLUGINS=1  # [not win]
     - affinder --help  # [not linux]
-    - "DISPLAY=localhost:1.0 xvfb-run -a affinder --help"  # [linux]
-    - "DISPLAY=localhost:1.0 xvfb-run -a python -c 'import affinder'"  # [linux]
+    - DISPLAY=localhost: 1.0 xvfb-run -a affinder --help  # [linux]
```

Comment:
Ah! I've dealt with this before but the error didn't seem familiar. But, this was in the bot's commit. I presume we should raise an issue there, but I can't find the repo where the bot lives? 🙏

Reply:
(Thank you btw!!! 🙏)

Reply:
You can open an issue here: https://github.com/regro/cf-scripts/issues

For the job still failing, it seems like you need to add explicitly PyQt as run dependency

Reply:
> For the job still failing, it seems like you need to add explicitly PyQt as run dependency

Ah, yes, that is because of a [recent change to the napari feedstock](https://github.com/conda-forge/napari-feedstock/pull/48) that I forgot about. 😅 Thank you! 🙏